File: Codigos estudiantes por lenguaje/Java/Cristian Maldonado/Matemática/GCD_LCM.java
public class GCD_LCM {

    static int GCD(int a, int b){
        return b == 0? a: GCD(b, a % b);
    }
    static int LCM(int a, int b){
        return a * (b / GCD(a, b));
    }
    
    public static void main(String[] args) {
        int a = 7, b = 13;
        System.out.println("GCD de a y b es: "+GCD(a, b));
        System.out.println("LCM de a y b es: "+LCM(a, b));
    }
}

File: Codigos estudiantes por lenguaje/C++/Cristian Maldonado/Matemática/GCD_LCM.cpp
#include <iostream>

using namespace std;

int GCD(int a, int b){
    return b == 0 ? a: GCD(b, a % b);
}
int LCM(int a, int b){
    return a * (b / GCD(a, b));
}

int main()
{
    ios_base::sync_with_stdio(false);cin.tie(NULL);
    int a, b;
    cin >> a >> b;
    cout << "GCD de A y B es: "<<GCD(a, b)<<endl;
    cout << "LCM de A y B es: "<<LCM(a, b)<<endl;
    return 0;
}
